web/src/components/PoisMobile.tsx

```diff
@@ -163,24 +177,14 @@ const PoisMobile = ({
             <StyledIcon icon={faArrowLeft} direction={direction} />
           </BackNavigation>
         )}
-        {/* To use geolocation in a development build you have to start the dev server with "yarn start --https" */}
-        <GeolocateControl
-          style={{
-            bottom: bottomActionSheetHeight,
-            position: 'absolute',
-            right: 0,
-          }}
-          positionOptions={{ enableHighAccuracy: true }}
-          trackUserLocation
-          position='bottom-right'
-        />
       </MapView>
       <BottomActionSheet
         title={!currentFeatureOnMap ? t('listTitle') : undefined}
         toolbar={toolbar}
         ref={sheetRef}
         setBottomActionSheetHeight={setBottomActionSheetHeight}
         direction={direction}>
+        <div ref={geolocatePosition} style={{ position: 'absolute', right: 10, top: -40 }} />
```

Comment:
Please use a styled-component. Also, this seems like it won't look correct for RTL